bowling/BowlingTests.cpp

```diff
@@ -0,0 +1,247 @@
+#include <catch2/catch_all.hpp>
+#include <algorithm>
+#include <vector>
+#include <iostream>
+
+SCENARIO("Testing strike score", "[int][string][strike]"){
+
+    GIVEN("Strike configuration"){
+    std::string givenConfiguration = "X";
+
+        WHEN("All 10 pins are hit"){
+            int score = 10;
```

Comment:
Some action is missing in WHEN, like `auto result = game.roll(givenConfiguration)`